src/@gliff-ai/upload/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module "@gliff-ai/upload";
```

Comment:
why do we need to do this? Why can't we just `npm install @gliff-ai/upload` and use it like we would any other module?

Reply:
I don't know. I've added that file to fix the error "Could not find a declaration file for module '@gliff-ai/upload'" (despite the typing files being in @gliff-ai/upload). I don't seem to be able to make it work without it.

Reply:
@cooper667 any ideas? Sounds like the types aren't being packaged properly?

Reply:
Rather than having the typings in a separate folder, you can put them in the root and not need to reference them in package.json (as long as there's an index.d.ts)

I tried this locally and it worked ok (in my IDE)

Reply:
Ok, i've moved the typing files in the root and removed "typings" reference in package.json, but seems to be getting an error in ui.tsx at line 413 (TS2604: JSX element type 'UploadImage' does not have any construct or call signatures.)

Reply:
I can't replicate this? I checked out this branch, did `rm -rf node_modules/ && npm i` then linting and running seems fine?